### PR TITLE
Feature/sort army list

### PR DESCRIPTION
**Pull request by roj42**
Adds a sort-by dropdown to the codex (alphbetical, points asc, points desc) and buttons to sort the army list (they obliterate the sort-dragging you did to sort 'em)

This was a request from a member of my group, all of us are using this tool. :)

### CODE REVIEW

File: src/components/ArmyList.vue
Comment (furka) [inline src/components/ArmyList.vue]: Adding these elements to the `ArmyList` causes the first item in the list to be bumped off the page.
image

It will also mess with the calculation for each unit's size based on points cost.
https://github.com/furka/40k-10th-list-builder/blob/master/src/components/ArmyList.vue#L11

Not sure what the best solution for this is

Comment (roj42) [reply src/components/ArmyList.vue]: Hmm. Lazy of me to miss that. The solution is probably some esoteric CSS.

I'll fix it.

Comment (roj42) [reply src/components/ArmyList.vue]: Okay. Please try again.

**Review by furka — APPROVED** [review]
looks good 👍

**Comment by roj42** [comment]
You need to merge it, as I don't have write access. :) How often do you update the hosted site, or is that someone else?

**Comment by furka** [comment]
I just need to publish a new version and it will auto-deploy, will do it after the merge

**Pull request merged**
